pkg/network/http/http_statkeeper.go

```diff
@@ -0,0 +1,62 @@
+// +build linux_bpf
+
+package http
+
+import (
+	"sync"
+
+	"github.com/DataDog/datadog-agent/pkg/process/util"
+)
+
+type Key struct {
+	SourceIP   util.Address
+	DestIP     util.Address
+	SourcePort uint16
+	DestPort   uint16
+}
```

Comment:
Just something to think about: For HTTP requests without keep alives each (src,dst) will map to a single request/path

Reply:
Very good point - I ended up taking `SourcePort` out of the key for this reason